simulate-event.js

```diff
@@ -272,6 +292,9 @@ module.exports = function (element, type, options) {
     cancelable: true
   }, result(eventOptions, eventType, element, type, options), options);
 
+  // Preserve the original or use the new options if none were given.
+  if (!original) original = options;
```

Comment:
I have an existing use case for playing back `keydown` events that were originally suppressed.